tests/queries/tests.py

```diff
@@ -2878,6 +2878,54 @@ def test_exclude_nullable_fields(self):
         )
 
 
+@skipUnlessDBFeature('supports_json_field')
+class ExcludeJSONFieldTests31894(TestCase):
```

Comment:
I'm not sure whether this should be here or in [tests/model_fields/test_jsonfield.py](https://github.com/django/django/blob/master/tests/model_fields/test_jsonfield.py).

Reply:
I'm not sure either :thinking: Probably it doesn't matter too much? Happy to move them if it seems important though!

Reply:
Please move these tests to `tests.model_fields.test_jsonfield.TestQuerying`. We don't need a separate class. Also, reuse existing objects is tests.